config/game_options.txt

```diff
@@ -398,6 +398,7 @@ ROUNDSTART_RACES human
 ROUNDSTART_RACES lizard
 #ROUNDSTART_RACES fly
 ROUNDSTART_RACES plasmaman
+ROUNDSTART_RACES dwarf
```

Comment:
Um. Could we have a discussion about this first in the terms of balance, whether they are deprived drunkards or not they are still small & hit really really hard, which could be a issue on the station. Self healing by ethanol is generally abusable when you've got raw ethanol on a tap or a stocked bar, as opposed to the moderate brewing processes of dwarves out on lavaland.